pkgs/development/embedded/stm32/stm32cubemx/default.nix

```
@@ -1,84 +1,115 @@
 { fdupes
+, buildFHSEnv
 , fetchzip
 , icoutils
 , imagemagick
 , jdk17
 , lib
 , makeDesktopItem
-, stdenv
+, stdenvNoCC
 }:
 
 let
   iconame = "STM32CubeMX";
-in
-stdenv.mkDerivation rec {
-  pname = "stm32cubemx";
-  version = "6.10.0";
+  pkg = stdenvNoCC.mkDerivation rec {
```

Cryptic abbreviated names are for 80-era C programmers.

Also, it is easy to insert or remove a single `s`.

```suggestion
  package = stdenvNoCC.mkDerivation rec {
```

---

Agree, done.